apps/wallet-mobile/src/components/Button/Button.tsx

```diff
@@ -130,4 +141,22 @@ const styles = StyleSheet.create({
     color: colors.buttonBackgroundBlue,
     fontWeight: '600',
   },
+  isCopying: {
+    position: 'absolute',
+    backgroundColor: '#000',
+    alignItems: 'center',
+    justifyContent: 'center',
+    top: -20,
+    alignSelf: 'center',
+    borderRadius: 4,
+    zIndex: 10
+  },
+  textCopy: {
+    color: 'white',
```

Comment:
all the colors needs to be applied as this for example: https://github.com/Emurgo/yoroi/blob/develop/apps/wallet-mobile/src/features/RampOnOff/common/ShowBuyBanner/BuyBannerSmall.tsx